src/sidebar/components/Annotation/AnnotationBody.js

```diff
@@ -18,6 +18,34 @@ import TagListItem from '../TagListItem';
  * @typedef {import("../../../types/config").SidebarSettings} SidebarSettings
  */
 
+/**
+ * Button to expand or collapse the annotation excerpt (content)
+ * @param {object} props
+ *   @param {string} [props.classes]
+ *   @param {(collapse:boolean) => void} props.setIsCollapsed
```

Comment:
For "obviously boolean" words like _collapsed_ I would usually omit the "is" from the setter name - ie. `setCollapsed`/`isCollapsed`. Maybe also from the current value prop itself.